<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotForm</class>
 <widget class="QWidget" name="PlotForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
  </layout>
  <action name="actionHorizontalZoom">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Icons/hZoomIcon</normaloff>:/Icons/hZoomIcon</iconset>
   </property>
   <property name="text">
    <string>Horizontal zoom</string>
   </property>
   <property name="toolTip">
    <string>Horizontal zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Icons/zoomOutIcon</normaloff>:/Icons/zoomOutIcon</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Icons/importTextData</normaloff>:/Icons/importTextData</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import text data to a specified file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSplineSmoothing">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Icons/splineSmoothing</normaloff>:/Icons/splineSmoothing</iconset>
   </property>
   <property name="text">
    <string>Spline smoothing</string>
   </property>
   <property name="toolTip">
    <string>Log spline smoothing</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAutoSplineSmoothing">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Icons/splineAutoSmoothing</normaloff>:/Icons/splineAutoSmoothing</iconset>
   </property>
   <property name="text">
    <string>Auto spline smoothing</string>
   </property>
   <property name="toolTip">
    <string>Spline smoothing without parameters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSavePicture">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Icons/importSaveIcon</normaloff>:/Icons/importSaveIcon</iconset>
   </property>
   <property name="text">
    <string>Save the picture</string>
   </property>
   <property name="toolTip">
    <string>Saves current view as an image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionDragXAxisLim">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/Icons/qtOpenHandIcon</normaloff>:/Icons/qtOpenHandIcon</iconset>
   </property>
   <property name="text">
    <string>Drag x axis limit</string>
   </property>
   <property name="toolTip">
    <string>Drags the current x-axes limits</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
